crates/primitives/src/lib.rs

```diff
@@ -56,6 +56,12 @@ pub use transaction::{
     TransactionSignedEcRecovered, TransactionSignedNoHash, TxHashOrNumber, TxType,
 };
 
+#[cfg(feature = "reth-codec")]
+pub use transaction::{
+    COMPACT_EXTENDED_IDENTIFIER_FLAG, COMPACT_IDENTIFIER_EIP1559, COMPACT_IDENTIFIER_EIP2930,
+    COMPACT_IDENTIFIER_LEGACY,
+};
+
```

Comment:
let's remove this export, and only make these accessible with `tx_type` module in the path. this, since the constant names don't contain tx type, so it can be a bit confusing what they refer to.

Reply:
https://github.com/paradigmxyz/reth/pull/12537/commits/ad5150ea788ceb5aff86cbb3994db34a6e652c6f